README.md

```diff
@@ -111,37 +109,37 @@ And that's it :D
    how [here](https://www.whatismybrowser.com/detect/what-is-my-local-ip-address))
 2. Open a web browser on your mobile device
 3. Access to the dashboard on your mobile: `http://<YOUR_LOCAL_IP>:3000` (e.g. `http://192.168.1.42:3000`)
-4. And voilà ! You can enable / disable the fullscreen on tap on the dashboard
+4. And voilà! You can enable / disable the fullscreen on tap on the dashboard
 
 > Note: Your mobile device AND your PC need to be on the same network
 
-> Tip: You can save a shortcut on your home screen to use it as an app like.
+> Tip: On an iOS device, you can add the webpage to your home screen like an app! When on the page, tap the Share button, navigate to "Add to home screen," wait ~2 seconds for it to verify, and confirm it. It will go to your home screen and show your neat dashboard shortcut!
```

Comment:
Can you write something for Android or add a generic message for pinning the page to the home screen?

Reply:
I'll try. I haven't had any experience with Android, but I assume there is a similar method

Reply:
Done!